docs/getting-started/first-runbook-run/create-runbook-projects.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Create Projects
+description: Step by step guide on how to create projects in Octopus Deploy.
+position: 20
+hideInThisSection: true
+---
+
+!include <create-projects>
```

Comment:
Minor comment - the use of this include in the context of a runbook ends referencing **deployment process** [like this](https://preprod.octopus.com/docs/getting-started/first-runbook-run/create-runbook-projects):

> You now have your first project and are ready to define your deployment process.
> The next step will create a runbook in the newly created project.

Reply:
The downside of shared content.  Removed the line about "define your deployment process"